pkg/image/registry/imagestreamimport/rest_test.go

```diff
@@ -198,3 +198,131 @@ func TestImportSuccessful(t *testing.T) {
 		}
 	}
 }
+
+func TestCheckImportFailure(t *testing.T) {
+	testNow := unversioned.Now()
+	nonzero := int64(100)
+	zero := int64(0)
+	tests := map[string]struct {
+		status         api.ImageImportStatus
+		stream         *api.ImageStream
+		tag            string
+		nextGeneration int64
+		now            unversioned.Time
```

Comment:
*nit*: why not use constants instead of two attributes above? They seem not to be overridden below.